<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeWidgetForm</class>
 <widget class="QWidget" name="ThemeWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="themeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="animatedLabel">
       <property name="text">
        <string>Animation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="animatedComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="legendLabel">
       <property name="text">
        <string>Legend:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="legendComboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="antialiasCheckBox">
       <property name="text">
        <string>Anti-aliasing</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QTableWidget" name="filterTable">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterComboBox">
       <item>
        <property name="text">
         <string>search</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lang</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>platform</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>length</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>released</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>votecount</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>has_anime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tag</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterOperatorComboBox">
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>!=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="filterItemCombobox"/>
     </item>
     <item>
      <widget class="QDateEdit" name="filterDateEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="filterAdd">
       <property name="text">
        <string>添加到过滤器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterDelete">
       <property name="text">
        <string>删除过滤器条目</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterApplyPushButton">
       <property name="text">
        <string>应用并抓取数据</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>themeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antialiasCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>legendComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animatedComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateUI()</slot>
 </slots>
</ui>
